### PR TITLE
feat: add mini-games page & mini-games routing

### PR DESCRIPTION
**Pull request by IlyaKozak**
1) add mini-games page (mini-games/):
image
2) add mini-games routing:
mini-games/sprint
mini-games/audiochallenge
mini-games/savannah
mini-games/custom-game

### CODE REVIEW

File: src/app/mini-games/pages/mini-games-page/mini-games-page.component.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<div class="flex-row">
```
Comment (rss-maksim) [inline src/app/mini-games/pages/mini-games-page/mini-games-page.component.html:1]: @suggestion
You could create array with games and iterate through it using ```*ngFor```

File: src/app/mini-games/pages/mini-games-page/mini-games-page.component.scss [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        margin: auto;
+        background: rgba(#344557, 0.85);
+    }
+
+    .flex-row {
+        display: flex;
+        justify-content: center;
+        flex-wrap: wrap;
+
+        .example-card {
```
Comment (rss-maksim) [inline src/app/mini-games/pages/mini-games-page/mini-games-page.component.scss:28]: @suggestion

It would be better to rename ```example-card``` class

**Review by rss-maksim — APPROVED** [review]

**Review by Diana-Valeeva — APPROVED** [review]

File: src/app/mini-games/pages/mini-games-page/mini-games-page.component.scss (hunk 2 above)
Comment (IlyaKozak) [reply src/app/mini-games/pages/mini-games-page/mini-games-page.component.scss]: done

File: src/app/mini-games/pages/mini-games-page/mini-games-page.component.html (hunk 1 above)
Comment (IlyaKozak) [reply src/app/mini-games/pages/mini-games-page/mini-games-page.component.html]: done

**Pull request merged**